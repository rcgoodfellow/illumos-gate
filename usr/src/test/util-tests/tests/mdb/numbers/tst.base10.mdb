0t23_24_25_26=E
0t123456=K
0t1________2_____________________3 = E
0t10$d
555_5555 = E
44_44_44_44_ = E
