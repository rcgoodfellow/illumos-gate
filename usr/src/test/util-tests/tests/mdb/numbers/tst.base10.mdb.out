                23242526             
                1e240           
                123                  
radix = 10 base ten
                5555555              
                44444444             
